core/android/src/main/java/com/posthog/reactnative/core/RNPostHogModule.kt

```diff
@@ -198,10 +198,11 @@ class RNPostHogModule(context: ReactApplicationContext): ReactContextBaseJavaMod
             )
 
     @ReactMethod
-    fun identify(distinctId: String, properties: ReadableMap?) =
+    fun identify(distinctId: String?, properties: ReadableMap?) =
             posthog.identify(
                     distinctId,
-                    Properties() from properties
+                    Properties() from properties,
+                    null
```

Comment:
I know this overloaded method [accepts Options](https://github.com/PostHog/posthog-android/blob/master/posthog/src/main/java/com/posthog/android/PostHog.java#L355) but given that iOS doesnt support options yet either in the react native library I felt that that was a different feature/PR